docs/tasks/run-application/configure-pdb.md

```diff
@@ -0,0 +1,161 @@
+  - Concern: do not terminate this application without talking to me.
+    - Possible Solution 1: Do not use a PDB and tolerate occasional downtime.
+    - Possible Solution 2: Set PDB with maxUnavailable=0.  Have understanding (outside of Kubernetes) that the cluster operator needs to consult before termination.
+      when the cluster operator contacts you, prepare for downtime, and then delete the to set maxUnavailable=1 to indicate readiness for disruption.  Recreate afterwards.
+- Multiple-instance Stateful application such as Consul, ZooKeeper, or etcd  
+  - Concern: Do not reduce number of instances below quorum, otherwise writes fail.
+    - Possible Solution 1: set maxUnavailable to 1 (works with varying scale of application).
+    - Possible Solution 2: set minAvailable to quorum-size (e.g. 3 when scale is 5).  (Allows more disruptions at once).
+- Restartable Batch Job:
+  - Concern: Job needs to complete in case of voluntary disruption
+    - Possible solution: Do not create a PDB.  The Job controller will create a replacement pod.
+
+## Specifying a PodDisruptionBudget
+
+A `PodDisruptionBudget` has three components, of which two must be specified: 
+
+* A label selector `selector` to specify the set of
+pods to which it applies. This is a required field.
+* `minAvailable` which is a description of the number of pods from that
+set that must still be available after the eviction, i.e. even in the absence
+of the evicted pod. `minAvailable` can be either an absolute number or a percentage.
+* `maxUnavailable` (available in Kubernetes 1.7 and higher) which is a description 
+of the number of pods from that set that can be unavailable after the eviction. 
+It can also be either an absolute number or a percentage.
+
+You can specify only one of `maxUnavailable` and `minAvailable` in a single `PodDisruptionBudget`. 
+`maxUnavailable` can only be used to control the eviction of pods 
+that have an associated controller managing them. In the examples below, "desired replicas"
+is the `scale` of the controller managing the pods being selected by the
+`PodDisruptionBudget`.
+
+Example 1: With a `minAvailable` of 5, evictions will be allowed as long as they leave behind
+5 or more healthy pods among those selected by the PodDisruptionBudget's `selector`.
+
+Example 2: With a `minAvailable` of 30%, evictions will be allowed as long as at least 30%
+of the number of desired replicas are healthy. 
+
+Example 3: With a `maxUnavailable` of 5, evictions will be allowed as long as there are at most 5
+unhealthy replicas among the total number of desired replicas.
+
+Example 4: With a `maxUnavailable` of 30%, evictions will be allowed as long as no more than 30% 
+of the desired replicas are unhealthy.
+
+In typical usage, a single budget would be used for a collection of pods managed by
+a controller—for example, the pods in a single ReplicaSet or StatefulSet. 
+
+Note that a disruption budget does not truly guarantee that the specified
```

Comment:
"Note: A disruption budget...."
We should address callouts in the style guide (note, caution, warning, etc)